.github/workflows/camunda-helm-integration.yml

```diff
@@ -38,15 +38,14 @@ jobs:
     runs-on: ubuntu-latest
     needs: [helm-deploy]
     if: ${{ always() }}
-    env:
-      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
-      SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
     steps:
       - id: slack-notify-failure
         name: Send failure slack notification
         uses: slackapi/slack-github-action@v2.0.0
         if: ${{ always() && needs.helm-deploy.result != 'success' }}
         with:
+          webhook: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
majority of changes is about https://github.com/slackapi/slack-github-action/releases/tag/v2.0.0#the-webhook-type-must-be-specified-for-incoming-webhooks